sync/settings.go

```diff
@@ -334,3 +298,48 @@ func ImportSettings(r *SettingsImportRequest) (*ImportResult, error) {
 
 	return result, nil
 }
+
+func checkSettings(settings map[string]any) (result *ImportResult, globalOnlySettingFound bool, err error) {
+	result = &ImportResult{}
+
+	// Validate config and gather some metadata.
+	var checked int
+	err = config.ForEachOption(func(option *config.Option) error {
+		// Check if any setting is set.
+		// TODO: Fix this - it only checks for global settings.
+		// if r.Reset && option.IsSetByUser() {
+		// 	result.ReplacesExisting = true
+		// }
+
```

Comment:
Is this something that needs to be sorted out before we release the import/export feature?